perfaware/sim86/shared/Sim8086.cpp

```diff
@@ -0,0 +1,147 @@
+#include <iostream>
+#include <fstream>
+#include <vector>
+#include <string>
+#include <sstream>
+#include "sim86_shared.h"
+#pragma comment (lib, "sim86_shared_debug.lib")
+#define ArrayCount(Array) (sizeof(Array) / sizeof((Array)[0]))
+
+constexpr int RegisterNumber = 15;
```

Comment:
It's not super useful here because you only have a single file, but in general, I would recommend getting in the habit of declaring all the global stuff (including functions) as static if it can be static. It kind of signals intent to some degree ("only intended for use in this translation unit"), but as soon as you have other files too, it can speed linking up.